app/main.js

```diff
@@ -167,17 +174,84 @@ function createWindow () {
     });
   }
 
+  win.removeAllListeners();
+
+  win.on('close', function(event) {
+    if (minimize) {
+      event.preventDefault();
+      win.hide();
+    }
+  })
+
   win.on('closed', () => {
     win = null
   })
 
+  win.on("hide", () => {
+    if (minimize) updateTray();
+  });
+
+  win.on("show", () => {
+    if (minimize) updateTray();
+  });
+
+  win.on("blur", () => {
+    globalShortcut.unregister('Alt+F4');
+  });
+
+  win.on("focus", () => {
+    globalShortcut.register('Alt+F4', () => safeQuit());
```

Comment:
This should be Ctrl/Cmd + Q as well.

Reply:
(Or if possible, detect OS setting for this, if one exists)

Reply:
Also needs to be available via a File menu in the app, I think.

Reply:
There is a predefined accelerator for this `CommandOrControl+Q`, which works on all OSes

And regarding the File Menu in the App, I already gave it a shot, and ran into some issues, I'll work on it. I would probably need to look at `module.exports`, as i need to "export" the `safeQuit()` function from `main.js` to `menu/main-menu.js`